src/main/jbake/content/blog/2015/07/01/hawkular-btm-0.1.0-demo.adoc

```diff
@@ -0,0 +1,180 @@
+= Monitoring a SwitchYard Application using Hawkular BTM 0.1.0.Final
+Gary Brown
+2015-07-01
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, btm, demo
+
+
+Hawkular BTM 0.1.0.Final is the first release of the Hawkular Business Management project, and therefore currently only provides a minimal level of functionality. However despite the early stage of the project, this post will show how a business transaction implemented using SwitchYard can be monitored across two separate applications using JMS and Web Services.
+
+This post assumes that the Hawkular BTM 0.1.0.Final distribution has been downloaded and unzipped, and the *_BTM_HOME_* environment variable setup, as described in the http://www.hawkular.org/docs/components/btm/index.html[user documentation]. The environment being monitored is *SwitchYard 2.0.0.Final* installed in *Wildfly 8.1.0.Final*.
+
+For this post, we will be deploying the SwitchYard _quickstarts/demos/multiApp_ example. Once SwitchYard has been installed in Wildfly, follow the instructions in the _multiApp/README.md_ to deploy the application into Wildfly. Once deployed, stop the Wildfly server.
+
+From a command window in the Wildfly _bin_ folder, run the following commands:
+
+[source,shell]
+----
+. $BTM_HOME/client/env.sh
+./standalone.sh -c standalone-full.xml
+----
+
+Simple as that! Now the Wildfly server, running the SwitchYard application, has been instrumented (using http://byteman.jboss.org/[ByteMan]) to capture some information relevant for Business Transaction Management. However before performing any transactions we need to startup the Business Transaction Management Server. Go to the BTM distribution's root folder and run:
+
+[source,shell]
+----
+bin/standalone.sh -Dkeycloak.import=standalone/configuration/hawkular-realm.json -Djboss.socket.binding.port-offset=100
```

Comment:
Break into two lines with a backslash at the end to prevent a possible horizontal scroll bar ?